core/src/main/resources/defaults/blueprints/streaming.bp

```diff
@@ -21,7 +21,6 @@
     },
     {
       "hbase-site" : {
-        "hbase.rootdir": "hdfs://hadoop-poc/apps/hbase/data",
         "zookeeper.recovery.retry" : "10"
       }
```

Comment:
these default properties don't have to be removed because they will be overridden by the filesystem configurations